code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -222,47 +222,50 @@
 	var/unlimited = 0
 	///Which species the resurected humanoid will be
 	var/applied_species = /datum/species/skeleton
+	///The outfit the resurected humanoid will wear
+	var/applied_outfit = /datum/outfit/roman
 
 /obj/item/necromantic_stone/unlimited
 	unlimited = 1
 
-/obj/item/necromantic_stone/attack(mob/living/carbon/human/M, mob/living/carbon/human/user)
-	if(!istype(M))
+/obj/item/necromantic_stone/attack(mob/living/carbon/human/target, mob/living/carbon/human/user)
+	if(!istype(target))
 		return ..()
 
-	if(!istype(user) || !user.can_perform_action(M))
+	if(!istype(user) || !user.can_perform_action(target))
 		return
 
-	if(M.stat != DEAD)
+	if(target.stat != DEAD)
 		to_chat(user, span_warning("This artifact can only affect the dead!"))
 		return
 
 	for(var/mob/dead/observer/ghost in GLOB.dead_mob_list) //excludes new players
-		if(ghost.mind && ghost.mind.current == M && ghost.client)  //the dead mobs list can contain clientless mobs
+		if(ghost.mind && ghost.mind.current == target && ghost.client)  //the dead mobs list can contain clientless mobs
 			ghost.reenter_corpse()
 			break
 
-	if(!M.mind || !M.client)
+	if(!target.mind || !target.client)
 		to_chat(user, span_warning("There is no soul connected to this body..."))
 		return
 
 	check_spooky()//clean out/refresh the list
 	if(spooky_scaries.len >= 3 && !unlimited)
 		to_chat(user, span_warning("This artifact can only affect three undead at a time!"))
 		return
-
-	M.set_species(applied_species, icon_update=0)
-	M.revive(ADMIN_HEAL_ALL)
-	spooky_scaries |= M
-	to_chat(M, "[span_userdanger("You have been revived by ")]<B>[user.real_name]!</B>")
-	to_chat(M, span_userdanger("[user.p_theyre(TRUE)] your master now, assist [user.p_them()] even if it costs you your new life!"))
+	if(applied_species)
+		target.set_species(applied_species, icon_update=0)
+	target.revive(ADMIN_HEAL_ALL)
+	spooky_scaries |= target
+	to_chat(target, "[span_userdanger("You have been revived by ")]<B>[user.real_name]!</B>")
```

Comment:
```suggestion
	to_chat(target, span_userdanger("You have been revived by <B>[user.real_name]</B>!")
```

this should work in a much cleaner way imo, may as well do it now since i have no clue when the next time this code is gonna be touched